cmd/errtrace/toolexec.go

```diff
@@ -10,6 +10,7 @@ import (
 	"path/filepath"
 	"runtime"
 	"runtime/debug"
+	"slices"
```

Comment:
We currently have 1.20 in the supported list.
Options: drop the 1.20, or don't use slices.Contains.
Your call. I don't have a preference.

Reply:
Kept 1.20 since it was simple enough to build a pre-1.21 slices contains.